### DISCUSSION TITLE
mylyn.org and ci.mylyn.org will be retired on March 15th

### DISCUSSION THREAD

**Discussion by BeckerFrank** [eclipse-mylyn/org.eclipse.mylyn#466] [Announcements]
@wimjongman, @ruspl-afed, @gnl42, @merks, @sddavis:

I got the following mail:

Hi Frank,

I'm forwarding this to you because I was unable to send it to [mylyn-dev@eclipse.org](mailto:mylyn-dev@eclipse.org).

Cheers,
Sam

From: Sam Davis
Sent: Friday, March 8, 2024 3:40 PM
To: Mylyn developer discussions <[mylyn-dev@eclipse.org](mailto:mylyn-dev@eclipse.org)>
Subject: [mylyn.org](http://mylyn.org/) and [ci.mylyn.org](http://ci.mylyn.org/) will be retired on March 15th
 
Hi,

As [previously announced](https://www.eclipse.org/lists//mylyn-dev/msg02496.html), [mylyn.org](http://mylyn.org/) and [ci.mylyn.org](http://ci.mylyn.org/) will be retired. This will happen on or after March 15th, in one week. All data will be permanently deleted, so if there is anything on the server that you need, please back it up now.

Cheers,
Sam

Sam Davis | Staff Software Engineer | Planview (Formerly Tasktop)
This message and any attached documents contain information of Planview, Inc., its affiliates, and/or third parties that may be confidential and/or privileged. If you are not the intended recipient, you may not read, copy, distribute, or use this information. If you have received this transmission in error, please notify the sender immediately by reply e-mail and then delete this message.

**Comment by BeckerFrank** [comment]
So this means that we no longer can run all our unit tests!

Actual I did not have a actual hardware that I can open to the internet to host the Multipass environment.

What can we do to continue with unit tests?

**Comment by wimjongman** [comment]
You can open a ticket with the webmasters and ask for resources. Can we run it on our build server?

**Reply by ruspl-afed** [reply]
Please open a ticket @BeckerFrank , you are the only expert in this field upon us and only you can formulate such a ticket well.

**Comment by BeckerFrank** [comment]
We allredy have a ticket. https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3432

We did not have a  https://wiki.eclipse.org/CBI#Additional_Resource_Packs or a sponsor so I did a test to share my own private server over a subdomain. 

This works but then I can not use this environment from my internal network to do the same tests. Also my hardware is to old but with a new Hardware it is possible. The other possibility is to get  a server from one cheap provider and there host the same services as mylyn.org.

Thoughts?

**Reply by ruspl-afed** [reply]
I don't think this should be your personal problem @BeckerFrank , let's see what Eclipse IDE WG will answer us (https://www.eclipse.org/lists/eclipse-ide-wg/msg00354.html)
If they are not interested, then our only option is to suspend integration tests until better times.

**Comment by gnl42** [comment]
@BeckerFrank Do you have any documentation on how you set up your local CI server?
I would like to do the same on my machine

**Reply by BeckerFrank** [reply]
Yes, I use Multipass and have included the sources and a README.md in https://github.com/eclipse-mylyn/org.eclipse.mylyn/tree/main/org.eclipse.mylyn.releng/multipass.

If you have issues or problems, feel free to ask.

Maybe we should open a Q&A for that topic.
